src/ui/components/services/storage.ts

```diff
@@ -0,0 +1,21 @@
+import Todo from '../models/todo';
```

Comment:
Run into next issue
`Error: The type of module 'services/storage' could not be identified`